src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -79,10 +94,14 @@ export const NavBarSchedule: FC<NavBarScheduleProps> = ({ isVisible }) => {
   const [selectedDayIndex, setSelectedDayIndex] = useState(0);
 
   const weekdays = useMemo(() => {
-    const today = startOfToday();
+    const formatDayLabel = (day: Date) => {
+      if (isToday(day)) return "Today";
+      if (isToday(firstDayOfSchedule)) return format(day, "E");
+      return format(day, "E, LLL d");
+    };
 
     return range(0, SCHEDULE_SHOW_DAYS_AHEAD).map((dayIndex) => {
-      const day = addDays(today, dayIndex);
+      const day = addDays(firstDayOfSchedule, dayIndex);
```

Comment:
See comment in this review related to this for more context.

```suggestion
      const day = addDays(firstDayOfSchedule, dayIndex);
      const formatDayLabel = (day: Date | number) => {
        if (isScheduleTimeshifted) {
          return format(day, "E, LLL d");
        } else {
          return dayIndex === 0 ? "Today" : format(day, "E");
        }
      }
```

Reply:
Potentially slightly more robust, instead of following the assumption that `dayIndex === 0` will imply today when not time shifted, we could be explicit about it, which might make the code easier to read.

```suggestion
      const day = addDays(firstDayOfSchedule, dayIndex);
      const formatDayLabel = (day: Date | number) => {
        if (isScheduleTimeshifted) {
          return format(day, "E, LLL d");
        } else {
          return isToday(day) ? "Today" : format(day, "E");
        }
      }
```

Reply:
done